fMRIVolume/scripts/OneStepResampling.sh

```diff
@@ -108,23 +137,28 @@ echo "date: `date`" >> $WD/log.txt
 echo " " >> $WD/log.txt
 
 
-########################################## DO WORK ########################################## 
+########################################## DO WORK ##########################################
+
+if [ `${FSLDIR}/bin/imtest ${BiasField}` -eq 0 ]; then
+  verbose_red_echo "---> Missing Bias Field image, creating a fake one"
+  ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 -add 1 ${BiasField}
+fi
```

Comment:
I guess this is fine, but under what situation would the BiasField be missing in the first place?

Reply:
This is a remnant of no-biasfield option when T2w image is missing. Now we made biasfield obligatory even if ony T1w image is present, so I can take this part out.

Reply:
That was in the PreFreeSurferPipeline.